app/ui/src/main/java/de/mm20/launcher2/ui/launcher/search/location/LocationItem.kt

```diff
@@ -477,7 +477,7 @@ fun LocationItem(
                                         Intent.ACTION_VIEW,
                                         Uri.parse("google.navigation:q=${location.latitude},${location.longitude}")
                                     ),
-                                )
+                                ).or(location.launch(context, null))
```

Comment:
Afaik `google.navigation:q=lat,lon` is the de facto standard for navigation intents and is supported by several navigation apps (i.e. OsmAnd, HERE We Go). Since there already are two "view on map" buttons (the map itself and the toolbar button), this seems pretty redundant to me and it would be better to remove the chip altogether when there is no navigation app installed.

Reply:
All I can say is that it does not work for me as an OrganicMaps user. Question then is how to determine if there is no app that can handle that intent?

Reply:
Alright, let's keep it then.

> Question then is how to determine if there is no app that can handle that intent?

You can use [`PackageManager.resolveActivity`](https://developer.android.com/reference/android/content/pm/PackageManager#resolveActivity(android.content.Intent,%20int)) to find an activity that can handle a given intent. It returns `null` if there's none.

Reply:
I've removed the chip now when there is no activity to resolve navigation intents.